external/llvm-project/mlir/lib/Conversion/AMDGPUToROCDL/AMDGPUToROCDL.cpp

```diff
@@ -187,7 +187,7 @@ struct RawBufferOpLowering : public ConvertOpToLLVMPattern<GpuOp> {
     //  none, 3 = either swizzles or testing against offset field) RDNA only
     // bits 30-31: Type (must be 0)
     uint32_t word3 = (7 << 12) | (4 << 15);
-    if (chipset.majorVersion == 10) {
+    if (chipset.majorVersion == 10 || chipset.majorVersion == 11) {
```

Comment:
Might as well swap that to `>= 10` at this point.

Reply:
Yes, why not!